### ISSUE TITLE
Add support for overriding the default prefix order used to choose the preferred identifier

### ISSUE THREAD

**Issue by gaurav** [TranslatorSRI/Babel#375]
We currently use the Biolink preferred prefix mappings (with some exceptions, particularly around conflation) but there isn't any way to customize it, either across Babel or for individual compendium builders.

@matentzn asked for this.

**Comment by matentzn** [comment]
Yes, this is of great importance. For example, biolink currently prefers to normalize phenotypes to UMLS over HPO, which is not very desirable in most open data normalisation efforts! Thanks!
